src/bootstrap-table.js

```diff
@@ -913,7 +913,11 @@ class BootstrapTable {
         return
       }
 
-      if (currentTarget === Utils.getSearchInput(this)[0] || $(currentTarget).hasClass('search-input')) {
+      if (
+        (currentTarget instanceof jQuery && currentTarget.is(Utils.getSearchInput(this))) ||
```

Comment:
What about to get the `Utils.getSearchInput(this)` in a variable and then just evaluate it in this if statement?

Reply:
I agree it's a micro optimization :-)